Mathlib/Analysis/Meromorphic/Basic.lean

```diff
@@ -52,15 +52,20 @@ theorem MeromorphicAt.eventually_eq_zero_or_eventually_ne_zero {f : 𝕜 → E}
 
 namespace MeromorphicAt
 
+variable
+  {ι : Type*} {s : Finset ι} {F : ι → 𝕜 → 𝕜} {G : ι → 𝕜 → E}
```

Comment:
I think putting these up here makes it harder to check the lemmas before; it's surprising to me that F and G have different tyoes, and it doesn't match how we handle f and g below.

Reply:
Moved the variables to the theorems where they are used.